python/cudf/cudf/tests/test_cuda_array_interface.py

```diff
@@ -186,3 +187,43 @@ def test_column_from_ephemeral_cupy_try_lose_reference():
     a.name = "b"
     b = cupy.asarray([1, 1, 1])  # noqa: F841
     assert_eq(pd.Series([1, 2, 3]), a.to_pandas())
+
+
+def test_cuda_array_interface_pytorch():
```

Comment:
You need to make this conditional on `torch` being installed, otherwise it should be skipped